frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -261,8 +262,19 @@ local footerTextGeneratorMap = {
     pages_left = function(footer)
         local symbol_type = footer.settings.item_prefix
         local prefix = symbol_prefix[symbol_type].pages_left
-        local left = footer.ui.toc:getChapterPagesLeft(footer.pageno)
-        return prefix .. " " .. (left or footer.ui.document:getTotalPagesLeft(footer.pageno))
+        local left = footer.ui.toc:getChapterPagesLeft(footer.pageno) or footer.ui.document:getTotalPagesLeft(footer.pageno)
+        return prefix .. " " .. left
+    end,
+    chapter_progress = function(footer)
+        local current = footer.ui.toc:getChapterPagesDone(footer.pageno)
+        -- We want a page number, not a page read count
+        if current then
+            current = current + 1
+        else
+            current = footer.pageno
+        end
+        local total = footer.ui.toc:getChapterPageCount(footer.pageno) or footer.pages
+        return current .. " ⁄⁄ " .. total
```

Comment:
Dunno how confusing this can be, but `//` is already used in page_progress when hasHiddenFlows().

Reply:
That's not *exactly* the same slash, I don't remember which one I settled on, but basically this one has *extreme* kerning, so you end up with the tow being very close together (the idea being it matches the double arrow of the "remaining pages in chapter" widget ;)).

Reply:
I'd be happy with changing the `//` with hidden flows to something else. Maybe `⌿`?

Reply:
![slashes](https://user-images.githubusercontent.com/111974/120931535-4364d900-c6f2-11eb-935d-edbb336d6ecb.png)

Right now, I'm using the *third* set (the two narrow slashes *very* close together). I was thinking of adding a hair space between them: that's the *first* set.

For comparison, a standard double-slash is on second place, and finally the ⌿.

I feel like it's different enough that we *can* keep the "double means chapter/flow" semantic hint between 'em (so, *notù touching the flows' double slash).